sound_main.py

```diff
@@ -0,0 +1,26 @@
+
+from selenium import webdriver
+import time
+
+driver = webdriver.Chrome('\chromedriver.exe')
+driver.get('https://www.audioknigi-online.com')

```

Comment:
@Volley97 Please define `argparse` argument with a url required argument, parse and use url from result args, instead of a hardcode